app-network/openvpn/autobuild/defines

```diff
@@ -1,6 +1,6 @@
 PKGNAME=openvpn
 PKGSEC=net
-PKGDEP="openssl lzo iproute2 systemd"
+PKGDEP="openssl lz4 lzo iproute2 systemd"
```

Comment:
Please specify lz4 support explicitly in AUTOTOOLS_AFTER.

Reply:
Updated :)